nodes.py

```diff
@@ -11,26 +11,25 @@ def pil_to_tens(image):
       image = image.convert('RGB')
     return ToTensor()(image).unsqueeze(0).permute(0, 2, 3, 1)
 
+def file_matches_filter(relative_file_path, filter_regex, extensions=None):
```

Comment:
I can pick this up in my PR

Reply:
In could be a constant in `ImageFromDirSelector`, yeah. But this this method is outside, it could also handle some other extensions, including none/any.